docs/source/access-data/tutorials/initial-setup.rst

```diff
@@ -3,194 +3,189 @@ Initial Setup
 
 **Learning Objectives:**
 
-- Complete the initial setup for data access
+Complete the initial setup required for data access:
 
-    1. Create a new GCP project.
-    2. Create a service account and configure authentication on a local machine.
-    3. Enable the APIs on your project and install related tools locally.
+    1. :ref:`Create a new GCP project.
+       <access-data/tutorials/initial-setup:Step 1: Create a GCP project>`
+    2. :ref:`Create a service account and configure authentication on a local machine.
+       <access-data/tutorials/initial-setup:Step 2: Configure authentication>`
+    3. :ref:`Enable the APIs on your project and install related tools locally.
+       <access-data/tutorials/initial-setup:Step 3: Enable and install APIs>`
 
 Our broker lives on `Google Cloud
 Platform <https://cloud.google.com/>`__ (GCP).
 To access the data you
```

Comment:
Minor grammatical error. I think this should be:

> To access the data, you need a free GCP project of your own with a configured service account authenticating your access to Google’s APIs.

Reply:
✅